services/venice-router/src/main/java/com/linkedin/venice/router/stats/RouteHttpRequestStats.java

```diff
@@ -54,15 +53,16 @@ public long getPendingRequestCount(String hostName) {
   }
 
   static class InternalHostStats extends AbstractVeniceStats {
-    private final Sensor pendingRequestCountSensor;
+    // private final Sensor pendingRequestCountSensor;
     private final Sensor unhealthyPendingQueueDuration;
     private final Sensor unhealthyPendingRateSensor;
     private AtomicLong pendingRequestCount;
 
     public InternalHostStats(MetricsRepository metricsRepository, String hostName) {
       super(metricsRepository, StatsUtils.convertHostnameToMetricName(hostName));
       pendingRequestCount = new AtomicLong();
-      pendingRequestCountSensor = registerSensor("pending_request_count", new Gauge(() -> pendingRequestCount.get()));
+      // pendingRequestCountSensor =
+      // registerSensor(new AsyncGauge((c, t) -> pendingRequestCount.get(), "pending_request_count"));
```

Comment:
If pendingRequestCountSensor is not used anywhere, let's remove it?

Reply:
Yup, it's removed.